main/acle.md

```diff
@@ -2524,8 +2524,8 @@ the [[cxxabi]](#cxxabi), and it is defined as follows:
 <vendor specific suffix> := `_` followed by token obtained from the tables below and prefixed with `M`
 ```
 
-If multiple features are requested then those shall be appended in increasing
-priority order and prefixed with `M`.
+If multiple features are requested then those shall be appended in lexicographic
+order and prefixed with `M`.
 
 For example:
```

Comment:
Could you please update the example below accordingly?

Reply:
Oh, yes, sorry I missed that!